Newcastle

chriswyatt1 opened the pull request:
---
name: New Config
about: A new cluster config
---

Please follow these steps before submitting your PR:

- [ ] If your PR is a work in progress, include `[WIP]` in its title
- [x] Your PR targets the `master` branch
- [x] You've included links to relevant issues, if any

Steps for adding a new config profile:

- [x] Add your custom config file to the `conf/` directory
- [x] Add your documentation file to the `docs/` directory
- [x] Add your custom profile to the `nfcore_custom.config` file in the top-level directory
- [x] Add your profile name to the `profile:` scope in `.github/workflows/main.yml`
- [ ] OPTIONAL: Add your custom profile path and GitHub user name to `.github/CODEOWNERS` (`**/<custom-profile>** @<github-username>`)

<!--
If you require/still waiting for a review, please feel free to request a review from @nf-core/maintainers

Please see uploading to`nf-core/configs` for more details:
https://github.com/nf-core/configs#uploading-to-nf-coreconfigs

Thank you for contributing to nf-core!
-->

chriswyatt1 committed:
Newcastle_dry_run

chriswyatt1 committed:
Update newcastle.config

chriswyatt1 committed:
Update newcastle.config

chriswyatt1 committed:
Instructions to download nextflow

chriswyatt1 committed:
Merge branch 'nf-core:master' into newcastle

chriswyatt1 committed:
Update Newcastle HPC cluster profile details

Add account flag

chriswyatt1 committed:
Clarify config parameters in newcastle.config

Added comments to clarify config parameters.

chriswyatt1 committed:
Update Newcastle config with new parameters and cleanup

chriswyatt1 committed:
limit queue size

chriswyatt1 committed:
Enhance Newcastle HPC configuration documentation

Updated Newcastle HPC configuration documentation with detailed usage notes, installation steps for Java and Nextflow, and troubleshooting tips.

chriswyatt1 committed:
Update container system references to Apptainer

Updated documentation to reflect the name change from Singularity to Apptainer and specified its availability on the comet system.

chriswyatt1 committed:
Add error strategy for task exit status handling as per nf-core

For non nf-core running pipelines

chriswyatt1 committed:
Update newcastle.md

chriswyatt1 committed:
Update example command for nextflow

chriswyatt1 committed:
Update newcastle.config

Added in correct resource limits section

jfy133 reviewed (approved):
Preemptive ✔️ once the relevant URL is added to the config description.

In the meantime @chriswyatt1 you could probably just point to the Newcastle University's homepage

chriswyatt1 committed:
Merge branch 'nf-core:master' into newcastle

chriswyatt1 committed:
Update newcastle.config

Add URL to Newcastle. 
Would be better to the actual docs of comet, but this is here for now.

chriswyatt1 committed:
Update newcastle.md

Add specific that its UK Newcastle University
Pseudonym: University of Newcastle upon Tyne

chriswyatt1 committed:
Update newcastle.md

Add link to official page in docs

chriswyatt1 committed:
Merge branch 'master' into newcastle
